discovery-provider/src/queries/queries.py

```diff
@@ -550,3 +551,11 @@ def get_creator_node_users():
         return api_helpers.success_response(users)
     except exceptions.ArgumentError as e:
         return api_helpers.error_response(str(e), 400)
+
+@bp.route("/l2/content_node", methods=("GET",))
+def get_l2_content_nodes():
+    try:
+        cnodes = get_l2_cnodes()
+        return api_helpers.success_response(cnodes)
+    except exceptions.ArgumentError as e:
+        return api_helpers.error_response(str(e), 400)
```

Comment:
maybe we should make this except block generalized to take any type of error?

Reply:
i don't feel strongly

Reply:
well I more so thinking that if an exception of not type `ArgumentError` was thrown, then the except block wouldn't be able to catch that error. then the dn wouldnt be able to serve an api error response